broadinstitute/single_cell_portal_core
# "My studies" refactor (SCP-4284)

This updates the My Studies dashboard to use react/API infrastructure.  The backend also no longer has an N+1 query issue, so it should load much faster for users with a large number of studies (previously, a separate DB query was made for every study to get the email address of the author)
(There are also two additional igv.css.map fixes added to routes.rb to reduce those errors in development)

![image](https://user-images.githubusercontent.com/2800795/169090511-ab9ca927-c1cc-440f-b7bc-b5318a6714dd.png)

TO TEST:
1. load the "My studies" page
2. confirm table loads, sorts, and filters
3. confirm study actions work as before.
